gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/gateway_query.proto

```diff
@@ -94,6 +94,7 @@ message Value {
   repeated double double_array = 13;
   repeated bool boolean_array = 14;
   map<string, string> string_map = 15;
+  string timestamp_unit = 16;
```

Comment:
Why do we need this in the value? The requester has access to the metadata anyway so should have this info (which saves us from repeating the unit in every response value - we already do this for all other units)

Reply:
Yeah that can also be done. But I will then need a different converter for Timestamp in gql, here https://github.com/hypertrace/hypertrace-core-graphql/pull/56#discussion_r617718178 since the convert method only has access to the Value object. 

> - we already do this for all other units)
where do we do this?

Reply:
My response from offline:
> So GQL will need changes to support timestamps as instants in attributes - right now it only works that way for top level APIs. The value converter needs to be changed to a biconverter I believe to take in the attribute metadata. Or we may need both the Converter and Biconverter flavors if there’s some places we call it where we don’t have that metadata at hand (probably just implement both methods and have an internal implementation, for example). The 400 error is due to the serialization side. Attributes right now use an UnknownScalar type, which needs to add support for instants - right now it treats them as plain objects.

Reply:
As to where we do this, I actually confused units and types. The units being joined in happens in the UI today - the types being joined in we do in GQL explorer results.